internal/api/documents.go

```diff
@@ -116,11 +125,19 @@ func DocumentHandler(
 			return
 		}
 
-		// Pass request off to the documents related resources handler if
-		// appropriate.
-		if isRelatedResourcesRequest {
+		// Pass request off to associated subcollection handler, if appropriate.
```

Comment:
Could we define what a sub collection is in a comment? would be helpful context.

Reply:
Done!